concepts/dates/introduction.md

```diff
@@ -0,0 +1,256 @@
+# Introduction
```

Comment:
Remember that the introduction should be a condensed version of the about file that only contains the most vital information. Let's get the about file sorted first but once we got that, think about what could be removed/shortened here. There should be just enough info to understand the concept and solve the exercise. We are not too strict about this atm but I think there are some quick fixes we could do here to make this more "introduction" like, e.g. remove the note on the 1.1.1970.